Documentation/op-guide/v2-migration.md

```diff
@@ -38,13 +38,17 @@ Second, migrate the v2 keys into v3 with the [migrate][migrate_command] (`ETCDCT
 
 Restart the etcd members and everything should just work.
 
+For etcd v3.3+, run `ETCDCTL_API=3 etcdctl endpoint hashkv --cluster` to ensure key-value stores are consistent post migration.
+
+**Warn**: When v2 store has expiring TTL keys and migrate command intends to preserve TTLs, it is possible that inconsistent data is migrated. TODO?
+
```

Comment:
s/it is possible that inconsistent data is migrated. TODO/migration may be inconsistent with the last committed v2 state when run on any member with a raft index less than the last leader's raft index./
?